stories/scrollbar/scrollbar.stories.js

```diff
@@ -0,0 +1,24 @@
+export default {
```

Comment:
I think it would be nice to explain why we have this component and when to use it. I guess it doesn't have to be mandatory?

Reply:
Updated the docs, please take a look.